src/main/java/com/crud/tasks/controller/TaskNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.crud.tasks.controller;
+
+public class TaskNotFoundException extends Exception {
```

Comment:
Hint (niewymagane w tym zadaniu): można tak stworzyć wyjątek, żeby właściwie mapował się na status HTTP (tutaj jeśli podajemy id nieistniejącego taska, zgodnie z REST powinniśmy otrzymać status `404 Not Found`). Zerknij tu: https://www.baeldung.com/spring-response-status-exception